projects/gloo/pkg/translator/utils_test.go

```diff
@@ -30,15 +29,17 @@ var _ = Describe("Utils", func() {
 	})
 
 	Context("UpstreamToClusterStatsName", func() {
-
 		DescribeTable("converting upstream to cluster stats name",
 			func(us *gloov1.Upstream, expectedStatsName string) {
 				statsName := translator.UpstreamToClusterStatsName(us)
 				Expect(statsName).To(Equal(expectedStatsName))
 			},
 			Entry("real kube upstream", createKubeUpstream(true, "name", "ns", "svcName", "svcNs", 123), "kube-upstream_name_ns_svcNs_svcName_123"),
 			Entry("fake kube upstream", createKubeUpstream(false, "name", "ns", "svcName", "svcNs", 123), "kube-svc_name_ns_svcNs_svcName_123"),
-			Entry("non-kube upstream", createStaticUpstream("name", "ns"), "name_ns"),
+			Entry("non-kube upstream", createStaticUpstream("name", "ns", nil), "name_ns"),
+			Entry("non-kube upstream", createStaticUpstream("name", "ns", map[string]string{
+				translator.ClusterStatNameLabel: translator.FormatClusterStatName("istio-se_", "usname", "usns", "svcns", "svcName", 123),
+			}), "istio-se_usname_usns_svcns_svcName_123"),
```

Comment:
@krisztianfekete  can you spot check this output would be correctly parsed

Reply:
This looks good to me!